mackerel-plugin-mongodb/lib/mongodb.go

```diff
@@ -105,6 +105,29 @@ var graphdef32 = map[string]mp.Graphs{
 	},
 }
 
+var graphdef36 = map[string]mp.Graphs{
```

Comment:
I think `graphdef36` and `metricPlace36` looks like no need because they are same value as `graphdef32` and `metricPlace32`.
Are there any reasons for separete them?

Reply:
exactly.
adapted version3.6 from function in version3.2